docs/examples/skanetrafiken/Readme.md

```diff
@@ -0,0 +1,161 @@
+
+### OSM
+
+We are using two OSM files:
+
+#### Sweden
+
+We download OSM data for whole Sweden, and then we clip northern part of it to reduce file size.
+
+```
+#!/bin/bash
+
+bashdir=$(dirname "$(readlink -f "$0")")
+
+mapHost=http://download.geofabrik.de/europe
+downloadFile=sweden-latest.osm.pbf
+now=$(date +%Y-%m-%dT%H%M%S)
+skanetrafikenFile=skanetrafiken.osm.pbf
+filterFile=skanetrafiken-filtered-$now.osm.pbf
+
+
+if [ -d /tmp/osm-skanetrafiken ]; then
+        echo "Delete existing /tmp/osm-skanetrafiken"
+        rm -rf /tmp/osm-skanetrafiken
+fi
+
+mkdir /tmp/osm-skanetrafiken
+cd /tmp/osm-skanetrafiken
+
+
+echo "Start download"
+wget $mapHost/$downloadFile
+
+echo "Cut skanetrafiken region"
+osmium extract --strategy complete_ways --bbox 11.7807,55.3,17.2162,57.7941 $downloadFile -o $skanetrafikenFile
+
+echo "Filter OSM data"
+osmium tags-filter $skanetrafikenFile w/highway w/public_transport=platform w/railway=platform w/park_ride=yes r/type=restriction -o $filterFile -f pbf,add_metadata=false
+
+mv $filterFile $bashdir
+
+echo "Created filtered OSM for skanetrafiken region ($filterFile)"
+
+rm -rf /tmp/osm-skanetrafiken
+```
+
+#### Denmark
+
+We download OSM data for whole Denmark, and then we clip southern part to reduce file size.
+
+```
+#!/bin/bash
+
+bashdir=$(dirname "$(readlink -f "$0")")
+
+mapHost=http://download.geofabrik.de/europe
+downloadFile=denmark-latest.osm.pbf
+now=$(date +%Y-%m-%dT%H%M%S)
+denmarkFile=denmark.osm.pbf
+filterFile=denmark-filtered-$now.osm.pbf
+
+if [ -d /tmp/osm-skanetrafiken ]; then
+        echo "Delete existing /tmp/osm-denmark"
+        rm -rf /tmp/osm-denmark
+fi
+
+mkdir /tmp/osm-denmark
+cd /tmp/osm-denmark
+
+echo "Start download"
+wget $mapHost/$downloadFile
+
+echo "Cut skanetrafiken region"
+osmium extract --strategy complete_ways --polygon oresund.json $downloadFile -o $denmarkFile
+
+echo "Filter OSM data"
+osmium tags-filter $denmarkFile w/highway w/public_transport=platform w/railway=platform w/park_ride=yes r/type=restriction -o $filterFile -f pbf,add_metadata=false
+
+mv $filterFile $bashdir
+
+echo "Created filtered OSM for skanetrafiken region ($filterFile)"
+
+rm -rf /tmp/osm-denmark
+```
+
+## Realtime
+
+The **Azure Service Bus** is used to propagate SIRI SX and ET realtime messages to OTP.
+This is solved through Siri Azure updaters that we had implemented in OTP. We have separate updaters
+for SIRI SX and ET.
+Those updaters are used to provide data for Swedish traffic (NeTEx). Right now, there is no
+connection
+to any realtime source for GTFS.
+
+Except for receiving messages from **Service Bus** we do also implement endpoints through which we
+fetch old messages on startup.
+We have two separate endpoint for respectively SX and ET history.
+The updater will first create new subscription on ServiceBus topic and then send request to the
+history endpoint.
+For Us, it takes some time get a response from the endpoint, so we tend to set timeout quite high.
```

Comment:
Suggestion, don't say "For us", say something like "It can take some time to fetch historic ET/SX messages"

Reply:
I've changed it now